test/integration/scheduler/queue_test.go

```diff
@@ -204,7 +204,7 @@ func TestCoreResourceEnqueue(t *testing.T) {
 				// However, due to preCheck, it's not requeueing pod2 to activeQ.
 				// It'll be fixed by the removal of preCheck in the future.
 				// https://github.com/kubernetes/kubernetes/issues/110175
-				node := st.MakeNode().Name("fake-node2").Capacity(map[v1.ResourceName]string{v1.ResourceCPU: "2"}).Taints([]v1.Taint{{Key: v1.TaintNodeNotReady, Effect: v1.TaintEffectNoSchedule}}).Obj()
+				node := st.MakeNode().Name("fake-node2").Label("node", "fake-node2").Capacity(map[v1.ResourceName]string{v1.ResourceCPU: "2"}).Taints([]v1.Taint{{Key: v1.TaintNodeNotReady, Effect: v1.TaintEffectNoSchedule}}).Obj()
 				if _, err := testCtx.ClientSet.CoreV1().Nodes().Create(testCtx.Ctx, st.MakeNode().Name("fake-node2").Capacity(map[v1.ResourceName]string{v1.ResourceCPU: "2"}).Taints([]v1.Taint{{Key: "foo", Effect: v1.TaintEffectNoSchedule}}).Obj(), metav1.CreateOptions{}); err != nil {
```

Comment:
🤦‍♂️🤦‍♂️ 

```suggestion
				if _, err := testCtx.ClientSet.CoreV1().Nodes().Create(testCtx.Ctx, node, metav1.CreateOptions{}); err != nil {
```

Reply:
Still not working. Why can't I reproduce this issue in my local environment?

Reply:
I'm not sure, is your local branch of this PR based on the latest master?

Reply:
Yes. I ran the integration test on my own branch which has rebased the lastest master. 
```
# make test-integration WHAT=./test/integration/scheduler/plugins/
+++ [0320 21:44:40] Checking etcd is on PATH
/root/work/kubernetes/third_party/etcd/etcd
+++ [0320 21:44:40] Starting etcd instance
etcd --advertise-client-urls http://127.0.0.1:2379 --data-dir /tmp/tmp.BNE9NlkLIF --listen-client-urls http://127.0.0.1:2379 --log-level=warn 2> "/dev/null" >/dev/null
Waiting for etcd to come up.
+++ [0320 21:44:41] On try 2, etcd: : {"health":"true","reason":""}
{"header":{"cluster_id":"14841639068965178418","member_id":"10276657743932975437","revision":"2","raft_term":"2"}}Periodically scraping etcd to /tmp/test.eUO7Ys/etcd-scrapes .
+++ [0320 21:44:41] Running integration test cases
make[1]: Entering directory '/root/work/kubernetes'
make[1]: warning: undefined variable '-*-command-variables-*-'
+++ [0320 21:44:41] Set GOMAXPROCS automatically to 8
+++ [0320 21:44:41] Running tests without code coverage
ok  	k8s.io/kubernetes/test/integration/scheduler/plugins	155.300s
make[1]: Leaving directory '/root/work/kubernetes'
+++ [0320 21:47:25] Cleaning up etcd
+++ [0320 21:47:25] Integration test cleanup complete
```

Reply:
It should be `make test-integration WHAT=./test/integration/scheduler`, right?

Reply:
Oops... My miskake. `CRTL+R` fooled me.